### PR TITLE
PR to Disable Tab Collapsing of the extension with a flag

### PR DESCRIPTION
**Pull request by BirdInFire**
This fix implement a flag to fix this issue : https://github.com/rafaelmardojai/firefox-gnome-theme/issues/587
and modify the readme to put the flag into the config.

### CODE REVIEW

**Comment by lazurit11** [comment]
I'm afraid this is not working right

image

**Comment by lazurit11** [comment]
It works, thanks! Also, it would be great if you could disable expanding-contracting showed on video below, or stop the page from following it (either would be fine), i reckon this will get annoying pretty fast

https://user-images.githubusercontent.com/62574650/230541032-89eeed8a-2c9f-4bdc-a87e-7291b7e6bebd.mp4

**Comment by BirdInFire** [comment]
> It works, thanks! Also, it would be great if you could disable expanding-contracting showed on video below, or stop the page from following it (either would be fine), i reckon this will get annoying pretty fast

I kinda knew the function but i didn't understood you asked for a full fix version, will check on fix that this evening

File: theme/extensions/tab-center-reborn.css [hunk 1]
```diff
@@ -117,3 +118,19 @@
         }
     }
 }
+
+@supports -moz-bool-pref("gnomeTheme.extensions.tabCenterReborn-always-open") {
```
Comment (rafaelmardojai) [inline theme/extensions/tab-center-reborn.css:122]: Let's call this `gnomeTheme.extensions.tabCenterReborn.alwaysOpen`.

**Review by rafaelmardojai — CHANGES_REQUESTED** [review]

File: theme/extensions/tab-center-reborn.css (hunk 1 above)
Comment (BirdInFire) [reply theme/extensions/tab-center-reborn.css:122]: No problem will fix his problem and change the name when i have time (i'm with friend today and tomorrow).

**Comment by BirdInFire** [comment]
So i have time tomorrow normally, to fix the sidebar and change the flag.

File: theme/extensions/tab-center-reborn.css (hunk 1 above)
Comment (BirdInFire) [reply theme/extensions/tab-center-reborn.css]: done with : https://github.com/rafaelmardojai/firefox-gnome-theme/pull/588/commits/a4428fc954eb2b7531f1f5b27a796f6117d833e7

**Comment by BirdInFire** [comment]
@lazurit11 totally fixed with this change : https://github.com/rafaelmardojai/firefox-gnome-theme/pull/588/commits/3a2fa9b54bda10a20d1bc45b530d28f1bc19d4a0

**Review by rafaelmardojai — APPROVED** [review]
LGTM.

**Pull request merged**